x-pack/plugins/uptime/public/routes.test.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// app.test.js
+import { screen } from '@testing-library/react';
+import { render } from './lib/helper/rtl_helpers';
+import { createMemoryHistory } from 'history';
+import React from 'react';
+import * as telemetry from './hooks/use_telemetry';
+import { MONITOR_ADD_ROUTE, MONITOR_EDIT_ROUTE } from '../common/constants';
+
+import '@testing-library/jest-dom';
+
+import { PageRouter } from './routes';
+
+jest.mock('../../../../src/plugins/kibana_react/public', () => ({
+  ...jest.requireActual('../../../../src/plugins/kibana_react/public'),
+  useKibana: () => ({
+    services: {
+      observability: {
+        navigation: {
+          PageTemplate: ({
+            children,
+            pageHeader,
+          }: {
+            children: React.ReactNode;
+            pageHeader: { pageTitle: React.ReactNode };
+          }) => (
+            <div>
+              <h1>{pageHeader.pageTitle}</h1>
+              {children}
+            </div>
+          ),
+        },
+      },
+    },
+  }),
+}));
```

Comment:
should this be combined with the context provider we have in the rtl_helper?

Reply:
Done, but I added a `@ts-ignore` because I didn't want to have to also mock all the rest of observability in one go.